tests/test_authsidecar.py

```diff
@@ -4,6 +4,12 @@
 import jmespath
 import yaml
 
+current_tag_level = "1.21.4"
```

Comment:
Why is this variable `current_tag_level`? That doesn't seem to make sense here.

Reply:
Updated to just explicitly declare the tag via fiat.